pages/agent/v3/tokens.md

```diff
@@ -14,6 +14,7 @@ The token is used by the Buildkite Agent's [start](/docs/agent/v3/cli-start#star
 It's recommended you use your platform's secret storage (such as the [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html)) to allow for easier rollover and management of your agent tokens.
 
 ## Creating tokens
+Note: There is a grace period of 10 minutes after Agent Token is created. During this period,  even if the page is reloaded the Token will still be visible. 
```

Comment:
Style tweaks:

```suggestion
There is a grace period of 10 minutes after you create a token. Even if you reload the page, the token remains visible during the grace period.
```

Reply:
@mbelton-buildkite Users were unsure why the Token still showed on the page even after they refreshed the page. Logged in and out. We want to clarify we have a grace period where the token is still visible after creation